### ISSUE TITLE
Share your website!

### ISSUE THREAD

**Issue by dwrolvink** [obsidian-html/obsidian-html#383]
I am compiling [a list of websites that use ObsidianHtml on the documentation site](https://obsidian-html.github.io/demonstrations/sites-using-obsidianhtml.html).

If you are using obsidianhtml and would not mind showing it to the world, please post it here. 

I'll add the list once I have at least a handful of examples.

**Comment by sa-aguilarv** [comment]
Hey! Could you please tell me which tool I can use to deploy the website to the world wide web? 
I tried with netlify but it couldn't build the site. I think it is because netlify only has build images for python 2.7 and 3.8 [source](https://github.com/netlify/build-image/blob/focal/included_software.md)
Thank you in advance for reading my message

**Comment by dwrolvink** [comment]
Hi sa-aguilarv. I just build the html on my own computer, into a git folder, and then do a git commit & push. Because it is a github pages repository, it will be hosted as a website. See also: https://pages.github.com/

- The repo: https://github.com/obsidian-html/obsidian-html.github.io
- The website: https://obsidian-html.github.io/

You could also have the building happen on github via https://github.com/features/actions

For netlify, python 3.10 should already be supported, this involves setting the PYTHON_VERSION variable, among possibly other settings, see here: https://docs.netlify.com/configure-builds/manage-dependencies/#python

Cloudflare pages also exists, but they support up to python 3.7 atm and python 3.9 is a real dependency, so that does not work.

**Comment by mhdrone** [comment]
Working on setting up mine now: https://mathnotes.mhdser.com/

**Comment by zubayrali** [comment]
Here's mine: https://zubayrrr.github.io, moved entirely to obsidian.md from dendron.so so it might look a bit clunky as of now.

**Comment by lowusage** [comment]
I've set mine online ! Will work on some theming later, but with a basic configuration it is already great ! 
 [yaourtiere.clypeata.fr](https://yaourtiere.clypeata.fr)

**Comment by erikwestra** [comment]
The first of my Obsidian-HTML based websites is finally starting to take shape: http://geyserlandbikepackingadventures.nz -- I created my own custom template to simplify the look, and so far I've written about half the content.  Hopefully I'll get the rest done in the next week or so.

**Comment by dwrolvink** [comment]
First version is live! :smile: 

https://obsidian-html.github.io/Demonstrations/Sites%20using%20ObsidianHtml.html

**Comment by camargomau** [comment]
I've finally had some time to properly configure obsidian-html and write a custom CSS file, so now I'm willing to share my website to the world. It's up on [notes.psycake.subspace.club](https://notes.psycake.subspace.club)!

**Comment by dwrolvink** [comment]
Very cool!

**Comment by sevmonster** [comment]
@dwrolvink Since the main doc site switched to slugified links, the page is now https://obsidian-html.github.io/demonstrations/sites-using-obsidianhtml.html. See also #618.

It looks like @yaourtiere might've switched to Hugo, might want to remove the page from the list.

My site: https://blog.sev.monster

**Comment by dwrolvink** [comment]
Ah! 
> "[ObsidianHtml](https://github.com/obsidian-html/obsidian-html) is honestly not that well put together a project, and I think it has many faults (...)" 

the gall! :rofl: 

Nah, fair enough lol.

Thanks for the pointers! Will add your site to the list.

**Comment by dwrolvink** [comment]
> "Social media is equivalent to the rotting of society’s collective brain."

agreed btw

**Comment by blacklightpy** [comment]
I have a site here: https://blacklightpy.github.io/vaultofknowledge, it's been up for a while.

But I wish to have some more personal discipline to organize my notes perfectly, so right now I consider it a bit of a mess, with even some of my personal todos and temporary brainstorms in it. Obsidian is now to me almost like Emacs is to Emacs people.

i'm very thankful for your project, it makes my site much more accessible than obsidian-zola which I used previously. It's not a bad project either, but it doesn't suit my system with nested folders. I might later on switch away from here for custom theming and layout, but if I can do it within this, I'll just share my theme and layout. But for now, this is really usable.

**Comment by dwrolvink** [comment]
I'm happy you like it!
Will add your site to the list

I'm assuming you've found this page already?
- https://obsidian-html.github.io/configurations/styling/edit-html-css-js.html

**Comment by blacklightpy** [comment]
Yes, but I'm a CSS noob as of now. So I don't know how much I can stretch it, for example to make the left pane look like Docusaurus / Sphinx book theme / PyData theme, etc.

**Comment by eurobots** [comment]
I started https://turm-lage.de/ as a website for my chess club. I put Obsidian in a Git repository. Closing obsidian triggers git commit and git push and obsidianhtml to our website. Now multiple club members can easily contribute to the website and as a bonus we have a shared Git repository and can use Obsidian for our written material.

**Comment by istarwyh** [comment]
I have a site here: https://xiaohui.cool/

I'm very thankful for your project!
